### PR TITLE
VZ-7432 PSR psrctl cleanup

### PR DESCRIPTION
**Pull request by pfmackin**
Some CLI cleanup for psrctl
Add support for namespace
Add more verbose output

### CODE REVIEW

File: tools/psr/psrctl/cmd/explain/explain.go [hunk 1]
```diff
@@ -15,24 +16,33 @@ import (
 
 const (
 	CommandName = "explain"
-	helpShort   = "Explain a PSR test scenario"
-	helpLong    = `The command 'explain' explains available scenarios that can be started`
+	helpShort   = "Explain a PSR scenario that can be started"
+	helpLong    = `The command 'explain' explains scenarios manifests that can be started.  The scenarios are represented by
```
Comment (mbenmclean) [inline tools/psr/psrctl/cmd/explain/explain.go:20]: "scanerios manifests that can be started" doesn't really make sense. Can you reword this.

File: tools/psr/psrctl/cmd/start/start.go [hunk 2]
```diff
@@ -14,17 +16,12 @@ import (
 const (
 	CommandName = "start"
 	helpShort   = "Start a PSR scenario"
-	helpLong    = `The command 'start' starts a PSR scenario`
+	helpLong    = `The command 'start' starts a PSR scenario in a namespace`
```
Comment (mbenmclean) [inline tools/psr/psrctl/cmd/start/start.go:19]: "in the specified namespace" maybe?

File: tools/psr/psrctl/cmd/list/list.go [hunk 3]
```diff
@@ -6,25 +6,26 @@ package list
...
-	helpLong    = `The command 'list' lists the PSR scenarios that are running in the cluster`
-	helpExample = `psrctl list `
-)
-
-const (
-	flagScenario       = "scenarioID"
-	flagsScenarioShort = "s"
-	flagScenarioHelp   = "specifies the scenario ID"
+	helpShort   = "List the running PSR scenarios"
+	helpLong    = `The command 'list' lists the PSR scenarios that are running in the specified namespace or the cluster`
```
Comment (stevshap) [inline tools/psr/psrctl/cmd/list/list.go:18]: specified namespace "of" the cluster?

File: tools/psr/psrctl/cmd/explain/explain.go (hunk 1 above)
Comment (pfmackin) [reply tools/psr/psrctl/cmd/explain/explain.go]: fixed

File: tools/psr/psrctl/cmd/list/list.go (hunk 3 above)
Comment (pfmackin) [reply tools/psr/psrctl/cmd/list/list.go]: fixed

File: tools/psr/psrctl/cmd/start/start.go (hunk 2 above)
Comment (pfmackin) [reply tools/psr/psrctl/cmd/start/start.go]: fixed

**Review by mbenmclean — APPROVED** [review]

**Review by stevshap — APPROVED** [review]

**Pull request merged**